### PR TITLE
Fixed SQS-Consumer span end time logging issue

### PR DESCRIPTION
**Pull request by willianpc**
So far, this has one of the two issues solved: The proper reported span end time.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+- Reporting correct SQS entry span time
```
Comment (basti1302) [inline CHANGELOG.md:4]: Actually, the span's _duration_ is what has been fixed, not the time 😝

File: packages/collector/test/tracing/cloud/aws/sqs/sqs-consumer.js [hunk 2]
```diff
@@ -36,10 +37,18 @@ const consumerApp = Consumer.create({
   queueUrl: queueURL,
   sqs,
   handleMessage: async message => {
+    const span = instana.currentSpan();
+    span.annotate('sdk.custom.tags.my_tag', 'group1');
```
Comment (basti1302) [inline packages/collector/test/tracing/cloud/aws/sqs/sqs-consumer.js:41]: AFAICT these two lines serve no purpose in the integration test and were only needed for a manual experiment. Maybe remove?

**Review by basti1302 — APPROVED** [review]
LGTM.

File: CHANGELOG.md (hunk 1 above)
Comment (willianpc) [reply CHANGELOG.md:4]: "ja ja" I will fix it :D ty!

Comment (basti1302) [reply CHANGELOG.md]: Ja, ja! https://www.youtube.com/watch?v=qrZxw5vdwRI&t=19s

**Pull request merged**